README.md

```diff
@@ -139,6 +141,16 @@
 * [Makes](https://github.com/fluidattacks/makes) - A Nix-based CI/CD pipeline framework for building, testing, and releasing projects in any language, from anywhere. 
 * [Standard](https://github.com/divnix/std) - An opinionated Nix Flakes framework to keep Nix code in large projects organized, accompanied by a friendly CLI/TUI optized for DevOps scenarios.
 
+## Continuous Integration
+
+* [Hercules CI](https://hercules-ci.com/) - Non free continuous integration service for Nix.
+* [Hydra](https://github.com/NixOS/hydra) - Official Nix-based continuous build system.
+* [Typhon CI](https://github.com/typhon-ci/typhon) - Nix-based continuous integration software inspired by Hydra.
+
+## Artificial Intelligence
+
+* [Nixified AI](https://nixified.ai/) - Making AI reproducible and easy to run.
```

Comment:
This should be in the `Overlays` section, since its purpose is to provide a set of packages, not to actually perform the Machine Learning.

(Could you even do machine learning with Nix? It's kind of counter to the whole "declarative" part given machine learning is entirely based on state)